app/channel_test.go

```diff
@@ -1554,6 +1554,30 @@ func TestAddUserToChannel(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestAddChannelPreviewer(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	user := model.User{Email: strings.ToLower(model.NewId()) + "success+test@example.com", Nickname: "Darth Vader", Username: "vader" + model.NewId(), Password: "passwd1", AuthService: ""}
+	ruser, _ := th.App.CreateUser(th.Context, &user)
```

Comment:
```suggestion
`ruser := th.CreateUser()`
```

The testing code has helpers to make things easier, it also does the error checks.